lupa-core/src/main/kotlin/org/jetbrains/research/lupa/kotlinAnalysis/util/JupyterInverter.kt

```diff
@@ -0,0 +1,144 @@
+package org.jetbrains.research.lupa
```

Comment:
It would be great to move this script into `org.jetbrains.research.lupa.kotlinAnalysis.util.python.jupiter` package. Also, please use the same package name with the file location